src/main/java/org/terasology/logic/health/BlockDamageAuthoritySystem.java

```diff
@@ -197,10 +197,10 @@ private void createBlockParticleEffect(BlockFamily family, Vector3f location) {
         final Stream<Vector2f> baseOffsets = Arrays.stream(BlockPart.sideValues()).map(blockAppearance::getTextureAtlasPos);
 
         return baseOffsets.flatMap(baseOffset ->
-                    IntStream.range(0, 8).boxed().map(i ->
+                IntStream.range(0, 8).boxed().map(i ->
```

Comment:
I tried using git reset -- HEAD command for this file 
but for some reason, this still shows up here.......😅

Reply:
Never mind, done. I realized I was comparing it to my fork...